pkg/cmd/kafka/consumergroup/describe/describe.go

```diff
@@ -204,7 +203,8 @@ func printConsumerGroupDetails(w io.Writer, consumerGroupData strimziadminclient
 	activeMembersCount := cgutil.GetActiveConsumersCount(consumers)
 	partitionsWithLagCount := cgutil.GetPartitionsWithLag(consumers)
 
-	fmt.Fprintln(w, color.Bold(localizer.MustLocalizeFromID("kafka.consumerGroup.describe.output.activeMembers")), activeMembersCount, "\t", color.Bold(localizer.MustLocalizeFromID("kafka.consumerGroup.describe.output.partitionsWithLag")), partitionsWithLagCount)
```

Comment:
Removed the bolding of text here because Command Prompt does not support ANSI colors. Powershell does and it worked fine though. Do we need to support both??

Reply:
![image](https://user-images.githubusercontent.com/11743717/115855408-1cdd2e00-a423-11eb-9971-d4fc07c0fb00.png)

Here's how it looks without bold titles. Looks just fine. I originally added bold to differentiate it from the values, but capitalising the title does an equally good job.

Reply:
Can you detect somehow?

Reply:
Yes - `if runtime.GOOS != "windows"`.

Reply:
I meant between powershell and cmd...

Reply:
Oh sorry, no I did not see a simple way..not using Go libraries anyway.

Reply:
That sucks. I guess we need to support both... I'm not sure how well adopted powershell actually is.

Reply:
Not sure either, it seems much better and has the same commands as Unix. Best to support both until we know for sure.